src/speakers.njk

```diff
@@ -0,0 +1,70 @@
+---
+layout: default-layout.njk
+title:  "Speakers"
+meta_description: Read more about the Vermont Code Camp speakers
+---
+
+
+<section class="main" >
+<div class="section-content">   
+
+
+<h1> {{title}} </h1>
+
+{% for speakerId, speaker in speakers %}
+
+    <article class="speaker article" id="{{ speaker.slug }}">
+        <div class="d-inline-flex headline ">
+            <h2 class="d-inline-flex" >
+                <a href="#{{ speaker.slug }}" >
+                    {{ speaker.fullName }} 
+                </a>
+            </h2>
+            {% for link in speaker.links %} 
+                {% if link.linkType == 'Twitter' %}
+                <h4 class="speaker-link twitter d-inline-flex">
+                    <a href="{{ link.url }}" class="d-inline-flex">
+                        <svg viewBox="0 0 512 512" >
+                            <use xlink:href="#twitter-path"></use>
```

Comment:
Cool technique to for reusing svg icons across the page without a separate network request!  Took me a little while to figure out where the *real* svgs were in the html output. 👍

Reply:
Yeah, I wanted to find a way to inline them for perf, but also be able to re-use them, well, for perf.

Is there anything that might make it more obvious where the actual icon paths are coming from?  I actually added the twitter one to the bottom of the page just for clarity.  We have it available anyway since the default layout has a twitter icon, but I didn't want to rely on the external dependency.

Reply:
If I had read to the bottom of the file and seen your comment it would have been much clearer to me, but instead of doing that I started searching in browser dev tools trying to figure it out 😄 .  I think it's plenty clear, I just wasn't familiar with the technique and thought it was cool once I *got it*.